packages/@ember/-internals/glimmer/lib/utils/to-bool.ts

```diff
@@ -1,9 +1,12 @@
+import { isHTMLSafe } from '@ember/-internals/glimmer';
 import { isArray } from '@ember/-internals/runtime';
 import { Opaque } from '@glimmer/interfaces';
 
 export default function toBool(predicate: Opaque): boolean {
   if (isArray(predicate)) {
     return (predicate as { length: number }).length !== 0;
+  } else if (isHTMLSafe(predicate)) {
+    return Boolean(String(predicate));
```

Comment:
So like this?

```suggestion
    return Boolean(predicate.toString());
```

Reply:
ya, thats what I was thinking, but I'm not really sure if thats the issue or not (was just grasping at straws a bit)